bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/OpenAPIUtils.java

```diff
@@ -56,7 +67,8 @@ public static Request requestBuilder(HttpClient httpClient, NanoleafControllerCo
 
         if (apiOperation.equals(API_ADD_USER)) {
             path = String.format("%s%s", API_V1_BASE_URL, apiOperation);
-        } else {
+        }
```

Comment:
Please join this with the else and/or apply formatter

Reply:
It seems that spotless does not 'repair' this, there should be no newline between '}' and 'else'

Reply:
It seems it did. There is no newline anymore now (or I have done it and forgot ;-) ). Resolved.